### ISSUE TITLE
"toBeAccessible()" test never passes

### ISSUE THREAD

**Issue by paul-balchin-ibm** [carbon-design-system/ibm-products#2857]
## What will this achieve?

Passing an accessibility unit test.

## How will success be measured?

A failing test will pass.

## Additional information

I could not pass the specific accessibility test below for the [Non Linear Reading](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2793/files#r1153521999) component.

Adding both of these extra tests passed, but did not solve the problem, and so were finally not included:
- Focus on body > tab > button has focus? Test passed. 
- Button is visible? Test passed. 

Debug
```jsx
<body>
  <div>
    <span> // <-- Component starts here.
      <button>
        Some text...
      </button>
```
Full report ([link](https://able.ibm.com/rules/archives/latest/doc/en-US/element_tabbable_visible.html#%7B%22message%22%3A%22Confirm%20the%20element%20should%20be%20tabbable%2C%20and%20is%20visible%20on%20the%20screen%20when%20it%20has%20keyboard%20focus%22%2C%22snippet%22%3A%22%3Cbutton%20class%3D%5C%22c4p--non-linear-reading--term-light%20c4p--non-linear-reading--term-light-closed%5C%22%20type%3D%5C%22button%5C%22%3E%22%2C%22value%22%3A%5B%22VIOLATION%22%2C%22POTENTIAL%22%5D%2C%22reasonId%22%3A%22potential_visible%22%2C%22ruleId%22%3A%22element_tabbable_visible%22%7D))
```
NonLinearReading › has no accessibility violations
Message: Confirm the element should be tabbable, and is visible on the screen when it has keyboard focus
Level: potentialviolation
Snippet: <button class="..." type="button">
```

Following the link, the documentation for this issue includes a "needs review" notification, but no details.

![image](https://user-images.githubusercontent.com/52505287/227326007-58e55a50-7b8f-49b8-996c-a20b3e780db7.png)

---

I have noticed that some other components `skip()` this test. See also `AboutModal.test.js`, `CreateFullPage.test.js`, and others.

**Comment by paul-balchin-ibm** [comment]
Typical. The problem was between the keyboard and the chair.

This problem was solved last week with a similar issue.

Until last week, I had been using the default React test command `yarn test path/to/COMPONENT_NAME.test.js` and this was always throwing errors.

As it has been recently pointed out, I should have been using IBM Products' specific `yarn test:c4p COMPONENT_NAME` instead. The tests pass using this command.

False alarm.

**Comment by elycheea** [comment]
Thank you, Paul. Classic PEBKAC. 😹
